imagetest/test_suites/packagevalidation/package_test.go

```diff
@@ -126,7 +141,7 @@ func TestGuestPackages(t *testing.T) {
 		},
 		{
 			name:       "google-compute-engine",
-			imagesSkip: []string{"sles", "suse"},
+			imagesSkip: []string{"sles", "suse", "cos"},
 		},
 		{
 			name:   "google-guest-configs",
```

Comment:
This package is the same as `google-compute-engine` just with a different name for sles and suse, it looks like you ship it the same way. Should probably be included here.